src/paperbot/api/routes/harvest.py

```
@@ -0,0 +1,448 @@
+    HarvestFinalResult,
+    HarvestPipeline,
+    HarvestProgress,
+)
+from paperbot.utils.logging_config import Logger, LogFiles, set_trace_id, clear_trace_id
+from paperbot.infrastructure.stores.paper_store import PaperStore, paper_to_dict
+
+router = APIRouter()
+
+# Lazy-initialized stores
+_paper_store: Optional[PaperStore] = None
+_research_store: Optional["SqlAlchemyResearchStore"] = None
```

Ruff (0.15.0)

[error] Undefined name `SqlAlchemyResearchStore`

(F821)